### PR TITLE
Warehouse crud

### PR DESCRIPTION
**Pull request by niiickm**
done:
-> warehouse item schema
-> warehouse item route/api
-> retrieving from warehouse and displaying on page
to do:
-> form validation
-> add item
-> edit item
-> liquidate
-> pull-out

PRing so warehouse schema/api can be used already

### CODE REVIEW

File: api/server.js [hunk 1]
```diff
@@ -46,6 +47,7 @@ app.use(morgan('tiny'));
 app.use(express.static('public')); 
 app.use('/api/suppliers', suppliers_routes);
 app.use('/api/chickens', chickenRoutes)
+app.use('/api/warehouse', warehouse_routes);
```
Comment (danjohnvelasco) [inline api/server.js]: change "/api/warehouse" --> "/warehouse"

File: client/src/views/Warehouse.vue [hunk 2]
```diff
@@ -340,24 +364,20 @@ export default {
     },
   },
 
-  created() {
-    this.initialize();
+  async mounted() {
+    const response = axios.get('/api/warehouse/');
```
Comment (danjohnvelasco) [inline client/src/views/Warehouse.vue]: Implement new changes (from ci-pipeline-update branch PR):

Add this to your <script> `const url = process.env.VUE_APP_API_URL;`

Use `${url}/warehouse/` instead.

Comment (danjohnvelasco) [reply client/src/views/Warehouse.vue]: Apply mo nalang din yung changes sa lahat ng /api

**Review by AxelAlba — DISMISSED** [review]
Next step: implement connection with operations db

**Review by AxelAlba — APPROVED** [review]

**Pull request merged**